### DISCUSSION TITLE
overlapScatterPlot output correlation coefficient and p values

### DISCUSSION THREAD

**Discussion by lauratwomey** [milaboratory/mixcr#1732] [Q&A] [answered]
Hi again!

I'm using overlapScatterPlot and it seems it can only output an image file (.png, .pdf)...
**is there a way to get the corr. coefficient R and p-value as output?** (.txt format or as part of the .log file)

Thanks!

**Comment by mizraelson** [comment, marked as answer]
Unfortunately, this function only exports a plot. We are currently working on our GUI software, which will have very broad plotting capabilities, including the export of statistical values in a table.

**Reply by lauratwomey** [reply]
Thank you so much!! Looking forward to it:)
